### DISCUSSION TITLE
Error when I try to install it via pip

### DISCUSSION THREAD

**Discussion by ashdxt** [perspective-dev/perspective#2056] [Q&A]
TypeError: expected str, bytes or os.PathLike object, not NoneType
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for perspective-python
  Building wheel for future (setup.py) ... done
  Created wheel for future: filename=future-0.18.2-py3-none-any.whl size=491104 sha256=11585fe6f14b664932c3793652ae5949bf5a11a8ac21d5989b9b4b394fb82c55
  Stored in directory: C:\Users\zas\AppData\Local\Temp\pip-ephem-wheel-cache-fn_fbe57\wheels\dc\16\09\eb08b4e34e6b638f113d2018cf0b22de1d8dca22a3a71873f7
Successfully built future
Failed to build perspective-python
ERROR: Could not build wheels for perspective-python, which is required to install pyproject.toml-based projects

I get the following error when I try to install it on python please advise

**Comment by timkpaine** [comment]
You're probably installing from the sdist and it's complaining about not finding `cmake` and/or compilers. 

see https://github.com/finos/perspective/discussions/1376#discussioncomment-573131

**Comment by texodus** [comment]
I've opened #2063 to fix this - I was personally burned by this today as well.
